app/src/main/java/org/wikiedufoundation/wikiedudashboard/ui/profile/view/ProfileFragment.kt

```diff
@@ -146,28 +111,34 @@ class ProfileFragment : Fragment(), ProfileContract.View, Toolbar.OnMenuItemClic
         llProfileParent.visibility = View.VISIBLE
         val profilePicUrl = Urls.BASE_URL + data?.userProfile?.profileImage
         Timber.d(profilePicUrl)
+
         if (data?.userProfile?.profileImage.isNullOrEmpty()) {
```

Comment:
Since Glide is being used, you can set a `placeholder` to it. And remove this conditional

Reply:
ok,working on them